README.md

```diff
@@ -102,7 +102,7 @@ Create a project. The following command will do some basic setup, create a super
 project-name, email, and password given here are specific to your local development environment. Ignore warnings
```

Comment:
I think this is incorrect now. Might also want to add a step to create a domain in the UI.

Reply:
Yeah, thanks. Decided not to add a step to create the domain because here's a confusing place to put it now, and later on once you start the server, the create a new project page is the first thing you see.